### PR TITLE
Create Day of the week indicator

### PR DESCRIPTION
**Pull request by korneliakobiela**
[Issue] #1105
[Problem] Day of the Week indicator doesn't exists
[Solution] Creating custom element of the indicator

Signed-off-by: Kornelia Kobiela <korneliak.95@gmail.com>

### CODE REVIEW

File: examples/mobile/ScheduleApp/js/day-indicator.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+		this.parseActiveDays().forEach((el) => {
+			this.days[el].classList.add("day-control-active");
+		})
+
+
+	}
+}
+
+
+customElements.define("day-indicator", DayIndicator);
```
Comment (HunseopJeong) [inline examples/mobile/ScheduleApp/js/day-indicator.js:71]: Question)
Is it working correctly on the tizen without transpiler or polyfill?

Comment (TomaszLukawskiSam) [reply examples/mobile/ScheduleApp/js/day-indicator.js:71]: I verified it on Tizen 5.5 emulator. Works good.

File: examples/mobile/ScheduleApp/css/style.css [hunk 2]
```diff
@@ -40,8 +40,45 @@
 .date-text {
 	font-size: 12px;
 	margin: auto 12px;
+	color: #909090;
```
Comment (TomaszLukawskiSam) [inline examples/mobile/ScheduleApp/css/style.css]: --text-secondary-color

File: examples/mobile/ScheduleApp/css/style.css [hunk 3]
```diff
@@ -40,8 +40,45 @@
...
+	height: 16px;
+	line-height: 16px;
+	margin: auto 12px;
+	font-size: 12px;
+}
+
+.day-control {
+	width: 13px;
+	height: 16px;
+	color: #909090;
```
Comment (TomaszLukawskiSam) [inline examples/mobile/ScheduleApp/css/style.css]: -text-secondary-color

File: examples/mobile/ScheduleApp/css/style.css [hunk 4]
```diff
@@ -40,8 +40,45 @@
...
+.day-control-active {
+	color: #0072DE;
+	font-family: "Roboto-Medium";
+}
+
+.day-control-active::before {
+	content: "";
+	width: 2.5px;
+	height: 2.5px;
+	background-color: #0072DE;
```
Comment (TomaszLukawskiSam) [inline examples/mobile/ScheduleApp/css/style.css]: --primary-dark-color

File: examples/mobile/ScheduleApp/css/style.css [hunk 5]
```diff
@@ -40,8 +40,45 @@
...
+	color: #909090;
+	font-size: 12px;
+	text-align: center;
+	font-family: 'Roboto-Regular';
+	display: inline-block;
+	
+}
+
+.day-control-active {
+	color: #0072DE;
```
Comment (TomaszLukawskiSam) [inline examples/mobile/ScheduleApp/css/style.css]: --primary-dark-color

**Review by TomaszLukawskiSam — CHANGES_REQUESTED** [review]
Color issues.

File: examples/mobile/ScheduleApp/css/style.css (hunk 2 above)
Comment (korneliakobiela) [reply examples/mobile/ScheduleApp/css/style.css]: Done!

File: examples/mobile/ScheduleApp/css/style.css (hunk 3 above)
Comment (korneliakobiela) [reply examples/mobile/ScheduleApp/css/style.css]: Done!

File: examples/mobile/ScheduleApp/css/style.css (hunk 4 above)
Comment (korneliakobiela) [reply examples/mobile/ScheduleApp/css/style.css]: Done!

File: examples/mobile/ScheduleApp/css/style.css (hunk 5 above)
Comment (korneliakobiela) [reply examples/mobile/ScheduleApp/css/style.css]: Done!

File: examples/mobile/ScheduleApp/css/style.css [hunk 6]
```diff
@@ -34,14 +34,51 @@
...
 
 .content-footer {
-	color: #909090;
+	color: var(--text-secondary-color);
 }
 
 .date-text {
 	font-size: 12px;
 	margin: auto 12px;
+	color: var(--text-secondary-color);
```
Comment (lslachciak) [inline examples/mobile/ScheduleApp/css/style.css:43]: text color of unselected items  in WebAppReference20313.pdf for dark theme is
#FDFDFD, in your case is #999999
I'll create separate issue for this

**Review by lslachciak — APPROVED** [review]
I don't see styling issues expect https://github.com/Samsung/TAU/issues/1112 which should be considered separately

File: examples/mobile/ScheduleApp/js/day-indicator.js [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+			this.days[i].innerText = this.daysLabels[i];
+			this.appendChild(this.days[i]);
+		}
+	}
+
+	parseActiveDays() {
+		return this.active ? this.active.split(" ").map((el) => +el) : [];
+	}
+
+	addActiveDays() {
```
Comment (TomaszLukawskiSam) [inline examples/mobile/ScheduleApp/js/day-indicator.js]: This method looks now like "updateActiveDays"

Comment (TomaszLukawskiSam) [reply examples/mobile/ScheduleApp/js/day-indicator.js]: Is this `protected' method ?

File: examples/mobile/ScheduleApp/js/day-indicator.js [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+	get active() {
+		return this.getAttribute("active");
+	}
+
+	set active(value) {
+		this.setAttribute("active", value);
+		this.updateActiveDays();
+	}
+
+	addDays() {
```
Comment (TomaszLukawskiSam) [inline examples/mobile/ScheduleApp/js/day-indicator.js]: I think this is good practice to add "_" prefix before protected methods, eg.
https://developers.google.com/web/fundamentals/web-components/examples/howto-checkbox

**Review by TomaszLukawskiSam — APPROVED** [review]
Looks good to me.

**Pull request merged**